eggdrop.conf

```diff
@@ -208,6 +208,12 @@ set switch-logfiles-at 300
 # date message at midnight, by setting this to 3.
 set quiet-save 0
 
+# If log flag d log user and sys times for triggered binds
```

Comment:
I think if you want debug, you get debug- let's remove these as config options and just implement the eval time line